src/fides/api/task/conditional_dependencies/evaluator.py

```diff
@@ -13,19 +14,75 @@
     Operator,
 )
 
+# Define operator methods for validation
 operator_methods = {
     Operator.exists: lambda a, _: a is not None,
     Operator.not_exists: lambda a, _: a is None,
     Operator.eq: py_operator.eq,
     Operator.neq: py_operator.ne,
-    Operator.lt: lambda a, b: a < b if a is not None else False,
-    Operator.lte: lambda a, b: a <= b if a is not None else False,
-    Operator.gt: lambda a, b: a > b if a is not None else False,
-    Operator.gte: lambda a, b: a >= b if a is not None else False,
+    Operator.lt: lambda a, b: (
+        a < b if a is not None and isinstance(a, numbers.Number) else False
+    ),
+    Operator.lte: lambda a, b: (
+        a <= b if a is not None and isinstance(a, numbers.Number) else False
+    ),
+    Operator.gt: lambda a, b: (
+        a > b if a is not None and isinstance(a, numbers.Number) else False
+    ),
+    Operator.gte: lambda a, b: (
+        a >= b if a is not None and isinstance(a, numbers.Number) else False
+    ),
     Operator.list_contains: lambda a, b: b in a if isinstance(a, list) else False,
-    Operator.not_in_list: lambda a, b: a not in b if isinstance(b, list) else True,
+    Operator.not_in_list: lambda a, b: b not in a if isinstance(a, list) else False,
```

Comment:
Is it possible for both the `a` and `b` params to `Operator.not_in_list` to be a list? Meaning you want to check if all the values in one list are present in another?